Python/gc.c

```diff
@@ -1697,7 +1696,7 @@ _PyGC_GetObjects(PyInterpreterState *interp, Py_ssize_t generation)
         }
     }
     else {
-        if (append_objects(result, GEN_HEAD(gcstate, generation))) {
+        if (append_objects(result, GEN_HEAD(gcstate, (int)generation))) {
```

Comment:
Why `generation` is not `int`? Changing the signature of `_PyGC_GetObjects()` may be more difficult, but what is the reason to use `Py_ssize_t` there?

Reply:
It was changed to `int` today (e28477f214276db941e715eebc8cdfb96c1207d9).